packages/orbit-components/src/Button/index.tsx

```diff
@@ -1,35 +1,89 @@
+  secondary:
+    "bg-button-secondary-background hover:bg-button-secondary-background-hover active:bg-button-secondary-background-active disabled:bg-button-secondary-background focus:bg-button-secondary-background-focus text-button-secondary-foreground focus:text-button-secondary-foreground-focus active:text-button-secondary-foreground-active hover:text-button-secondary-foreground-hover disabled:text-button-secondary-foreground focus:shadow-ink-normal/30 active:shadow-button-active-pale",
+  critical:
+    "bg-button-critical-background hover:bg-button-critical-background-hover active:bg-button-critical-background-active disabled:bg-button-critical-background focus:bg-button-critical-background-focus text-button-critical-foreground focus:text-button-critical-foreground-focus active:text-button-critical-foreground-active hover:text-button-critical-foreground-hover disabled:text-button-critical-foreground focus:shadow-red-normal/50 active:shadow-button-active",
+  primarySubtle:
+    "bg-button-primary-subtle-background hover:bg-button-primary-subtle-background-hover active:bg-button-primary-subtle-active-background disabled:bg-button-primary-subtle-background focus:bg-button-primary-subtle-background-focus text-button-primary-subtle-foreground focus:text-button-primary-subtle-foreground-focus active:text-button-primary-subtle-foreground-active hover:text-button-primary-subtle-foreground-hover disabled:text-button-primary-subtle-foreground focus:shadow-white-active/50 active:shadow-button-active-pale",
+  criticalSubtle:
+    "bg-button-critical-subtle-background hover:bg-button-critical-subtle-background-hover active:bg-button-critical-subtle-active-background disabled:bg-button-critical-subtle-background focus:bg-button-critical-subtle-background-focus text-button-critical-subtle-foreground focus:text-button-critical-subtle-foreground-focus active:text-button-critical-subtle-foreground-active hover:text-button-critical-subtle-foreground-hover disabled:text-button-critical-subtle-foreground focus:shadow-product-normal/50 active:shadow-button-active-pale",
+  white:
+    "bg-button-white-background hover:bg-button-white-background-hover active:bg-button-white-background-active disabled:bg-button-white-background focus:bg-button-white-background-focus text-button-white-foreground focus:text-button-white-foreground-focus active:text-button-white-foreground-active hover:text-button-white-foreground-hover disabled:text-button-white-foreground focus:shadow-red-normal/50 active:shadow-button-active-pale",
+  bundleBasic:
```

Comment:
It is applying `background-color` css property, but it has to be `background-image` because that is how gradients are applied to background. We have two options, the correct/difficult and the quickest but hacky/buggy/quickest: 

**Correct/difficult**: on the tokens definition and tailwind config, bundle tokens should not be treated as colors, but as images 🤷‍♂️ they won't work as colors anywhere else anyway, so having them defined as that is just wrong

**Hacky/buggy/quickest**: classes for bundle buttons must be `[background-image:button-bundle-basic-background]` (and so on)

I very much prefer the first one, since we're not on a rush for this component and it's an issue we need to address anytime soon

Reply:
I had a third idea - create two separate tokens, gradient from and gradient to, for each tier. then use this with the native Tailwind gradient API

do you think this'd work?

Reply:
@oreqizer We have available in the palette two basic bundle colors, this will work, but unfortunately, you will have to create a lot of new tokens, as it should also have hover/active variants and two tokens for each, meaning you will need to create 12 new tokens, so I would prefer the fix with background-image :)

Reply:
@oreqizer Yes, that will work, we can do it that way, in addition to our bundle tokens, and create more granular ones. You can create them inside *foundation/global/palette/gradients*

Reply:
solved it by specifying these colors as `backgroundImage` in the TW theme